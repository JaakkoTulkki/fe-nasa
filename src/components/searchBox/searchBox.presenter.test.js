import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import React from 'react'
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import SearchBox from './searchBox.presenter'

test('SearchBox renders', () => {
  const wrapper = shallow(<SearchBox
    searchTerm="space chicken"
    title="search title"
    errorMessage="something went wrong...."
    image
    audio={false}
  />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
