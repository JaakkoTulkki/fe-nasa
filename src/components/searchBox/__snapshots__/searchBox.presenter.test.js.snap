// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox renders 1`] = `
<div
  className="search_box"
>
  <h2>
    search title
  </h2>
  <div>
    <form>
      <input
        name="q"
        placeholder="Search...."
        type="text"
        value="space chicken"
      />
      <input
        onClick={[Function]}
        type="submit"
        value="Search"
      />
      <br />
      <input
        checked={true}
        name="image"
        onChange={[Function]}
        type="checkbox"
      />
      Image
      <input
        checked={false}
        name="audio"
        onChange={[Function]}
        type="checkbox"
      />
      Audio
    </form>
    <p
      className="search_box__error_message"
    >
      something went wrong....
    </p>
  </div>
</div>
`;
